Microsoft/WebTemplateStudio
# replace bootstrap with bootstrap vue

This is to address one of the Vue feedback: using Bootstrap-Vue. 

The following Bootstrap should be replaced by Bootstrap-Vue Components: 
- [x] [Breadcrumb](https://bootstrap-vue.js.org/docs/components/breadcrumb)

- [x] [Layout](https://bootstrap-vue.js.org/docs/components/layout): Grid should use _stacked to horizontal_ Bootstrap-Vue layout
- [x] [NavBar](https://bootstrap-vue.js.org/docs/components/navbar)

How to test:
1. you are able to generate a Vue project with 4 pages selected, the project should be able to start
2. the generated code should have changes contained from this PR. More details of Bootstrap Vue can be found in the links.
 
close #1035
